client/src/app/core/repositories/users/user-repository.service.ts

```diff
@@ -198,13 +246,47 @@ export class UserRepositoryService extends BaseRepositoryWithActiveMeeting<ViewU
      *
      * @param user The user to update
      * @param password The password to set
-     * @param updateDefaultPassword Control, if the default password should be updated.
+     * @param setAsDefault Control, if the default password should be updated. Defaults to `true`.
+     */
+    public async setPassword(user: ViewUser, password: string, setAsDefault: boolean = true): Promise<void> {
```

Comment:
Reagarding passwor handling: You opened an issue for a new action. I think it is not necessary. Please look closely into the email for user actions. There are all actions and their use cases described.